UnchainedLauncher.GUI/ViewModels/MainWindowViewModel.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnchainedLauncher.GUI.ViewModels {
+    public partial class MainWindowViewModel : INotifyPropertyChanged, IDisposable {
+        public LauncherViewModel LauncherViewModel { get; }
+        public ModListViewModel ModListViewModel { get; }
+        public SettingsViewModel SettingsViewModel { get; }
+        public ServerLauncherViewModel ServerLauncherViewModel { get; }
+        public ServersViewModel ServersViewModel { get; }
+
+        public MainWindowViewModel(): this(
+            new LauncherViewModel(), 
+            new ModListViewModel(),
+            new SettingsViewModel(),
+            new ServerLauncherViewModel(),
+            new ServersViewModel()
+        ) { }
```

Comment:
These default constructors are unusable in a deployed env. I'll remove them soon. I made them because I thought it was the only way to fix intellisense. I now know a better way